.github/workflows/prepare_release.yml

```
@@ -14,15 +14,20 @@ env:
 jobs:
   prepare-release:
     runs-on: ubuntu-latest
+    name: Prepare
     permissions:
       contents: write
       pull-requests: write
+    outputs:
+      branch_name: ${{ steps.prepare-release.outputs.BRANCH_NAME }}
+      release_commit_id: ${{ steps.prepare-release.outputs.RELEASE_COMMIT_ID }}
+      release_version: ${{ steps.get-release-versions.outputs.RELEASE_VERSION }}
+      new_snapshot: ${{ steps.get-release-versions.outputs.NEW_SNAPSHOT }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-        with:
-          ref: 'main'
-          fetch-depth: '0'
+        # with:
+        #   ref: 'main'
```

This should be uncommented before merging this PR, right?

```suggestion
        with:
          ref: 'main'
```